src/EnergyPlus/DataGenerators.hh

```diff
@@ -97,21 +97,21 @@ namespace DataGenerators {
 
     enum class ConstituentMode
     {
-        Unassigned,
+        Unassigned = -1,
         RegularAir,
         UserDefinedConstituents
     };
 
     enum class FuelTemperatureMode
     {
-        Unassigned,
+        Unassigned = -1,
         FuelInTempFromNode,
         FuelInTempSchedule
     };
 
     enum class WaterTemperatureMode
     {
-        Unassigned,
+        Unassigned = -1,
         WaterInReformMains,
```

Comment:
`WaterInReform` (whatever that means) should probably be part of the type name and removed from names of individual type values.